lib/exercism/github.rb

```diff
@@ -35,4 +35,11 @@ def self.authenticate(code, client_id, client_secret)
   def self.user_agent
     'github.com:exercism/exercism.io'
   end
+
+  def self.configure_octokit
+    Octokit.configure do |c|
+      c.login = 'SaiPramati'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.